package.json

```diff
@@ -1,51 +1,50 @@
 {
   "name": "@salesforce/plugin-org",
   "description": "Commands to interact with Salesforce orgs",
-  "version": "2.11.8",
+  "version": "3.0.0",
   "author": "Salesforce",
-  "main": "lib/index.js",
   "bugs": "https://github.com/forcedotcom/cli/issues",
   "dependencies": {
-    "@oclif/core": "^2.15.0",
+    "@oclif/core": "^3.10.6",
     "@salesforce/core": "^5.3.18",
     "@salesforce/kit": "^3.0.15",
-    "@salesforce/sf-plugins-core": "^3.1.28",
+    "@salesforce/sf-plugins-core": "^4.0.0",
     "@salesforce/source-deploy-retrieve": "^9.8.4",
-    "open": "^8.4.2",
-    "tslib": "^2"
+    "open": "^8.4.2"
   },
   "devDependencies": {
-    "@oclif/plugin-command-snapshot": "^4.0.14",
-    "@salesforce/cli-plugins-testkit": "^4.4.7",
+    "@oclif/plugin-command-snapshot": "^5.0.1",
+    "@salesforce/cli-plugins-testkit": "^5.0.2",
     "@salesforce/dev-scripts": "^6.0.3",
     "@salesforce/plugin-command-reference": "^3.0.46",
     "@salesforce/ts-sinon": "1.4.19",
-    "@swc/core": "1.3.39",
     "@types/shelljs": "^0.8.14",
     "eslint-plugin-sf-plugin": "^1.16.13",
     "moment": "^2.29.1",
-    "oclif": "^3.17.2",
+    "oclif": "^4.0.3",
     "shelljs": "^0.8.5",
```

Comment:
sandboxNut is the only thing using shelljs, just to exec things that *could* be done with execCmd and `cli:'sf'`

It'd still be using shelljs via testkit.execCmd, but could remove devDep and types for shelljs.